### ISSUE TITLE
Transcription filtering

### ISSUE THREAD

**Issue by vadimkantorov** [MoonshotAI/Kimi-Audio#27]
Hi!

Do you do any transcription filtering? E.g. whisper can hallucinate, especially on noisy audio. Do you do any confidence or agreement-based filtering?

Do you do any protection during model pre-training? E.g. how do you determine samples with bad reference transcription / skip bad gradients / skip bad samples in general et cetera?

Thank you :)
